modules/rhel-compute-updating.adoc

```diff
@@ -9,7 +9,7 @@ After you update your cluster, you must update the {op-system-base-full} compute
 
 [IMPORTANT]
 ====
-Because only {op-system-base-full} version 7.9 or later is supported for worker (compute) machines, you must not upgrade the {op-system-base} worker machines to version 8.
+Only {op-system-base-full} version 7.9 and version 8.4 is supported for worker (compute) machines, you must not upgrade the {op-system-base} worker machines to version 8.
```

Comment:
hi @kelbrown20 , some suggestions:
Maybe we can say more specifically like:
`{op-system-base-full} version 7.9 and version 8.4 is supported for {op-system-base} worker (compute) machines.`

And drop `you must not upgrade the {op-system-base} worker machines to version 8. `, because we had another note in Line19:
`You cannot upgrade {op-system-base} 7 compute machines to {op-system-base} 8.`